tensorflow/lite/micro/kernels/micro_ops.h

```diff
@@ -81,6 +81,7 @@ TfLiteRegistration Register_SUB();
 TfLiteRegistration Register_SVDF();
 TfLiteRegistration Register_TRANSPOSE();
 TfLiteRegistration Register_TRANSPOSE_CONV();
+TfLiteRegistration Register_UNIDIRECTIONAL_SEQUENCE_LSTM();
```

Comment:
nit: maybe add a todo here. (xtensa version needs to change b/c this change?)

Reply:
Done.